docs/ldml/tr35.md

```diff
@@ -3647,40 +3647,10 @@ and are included below the !ELEMENT or !ATTLIST line that they apply to. The cur
 | `<!--@DEPRECATED: attribute-value1, attribute-value2-->` | The attribute values are deprecated, and should not be used. Spaces between tokens are not significant. |
-| `<!--@MATCH:{attribute value constraint}-->` | Requires the attribute value to match the constraint. |
 | `<!--@TECHPREVIEW-->` | The element is a technical preview of a feature and may be changed or removed at any time. |
+| `<!--@MATCH:{attribute value constraint}-->` | Requires the attribute value to match the constraint. |
 
-There is additional information in the attributeValueValidity.xml file that is used internally for testing. For example, the following line indicates that the 'currency' element in the ldml dtd must have values from the bcp47 'cu' type.
-
-```xml
-<attributeValues dtds='ldml' elements='currency' attributes='type'>$_bcp47_cu</attributeValues>
-```
-
-The element values may be literals, regular expressions, or variables (some of which are set programmatically according to other CLDR data, such as the above). However, the information at this point does not cover all attribute values, is used only for testing, and should not be used in implementations since the structure may change without notice.
-
-#### <a name="match_expressions" href="#match_expressions">Attribute Value Constraints</a>
-
-The following are constraints on the attribute values. Note: in future versions, the format may change, and/or the constraints may be tightened.
-
-| Constraint                | Comments |
-| ------------------------- | -------- |
-| any                       | any string value |
-| any/TODO                  | placeholder for future constraints |
-| bcp47/anykey              | any bcp47 key or tkey |
-| bcp47/anyvalue            | any bcp47 value (type) or tvalue |
-| literal/\{literal values} | comma separated |
-| regex/\{regex expression} | valid regex expression |
-| bcp47/\{key or tkey}      | matches possible values for that key or tkey |
-| metazone                  | valid metazone |
-| range/\{start_number~{end_number}} | number between (inclusive) start and end |
-| time/\{time or date or date-time pattern} | eg HH:mm |
-| unicodeset/\{unicodeset pattern} | valid unicodeset |
-| validity/\{field}         | currency, language, locale, region, script, subdivision, short-unit, unit, variant<br/>The field can be qualified by particular enums, such as:<br/>`validity/unit/regular deprecated`: matches only _deprecated_ and _regular_<br/>`validity/unit/!deprecated`: matches all but _deprecated_ |
-| version                   | 1 to 4 digit field version, such as 35.3.9 |
-| set/\{match}              | set of elements that match \{match} |
-| or/\{match1}\|\|\{match2}…  | matches at least one of \{match1}, etc |
-
-
+Because they are intended for internal use in CLDR tooling, the {attribute value constraints} are described in [DTD Attribute Value Constraints](https://cldr.unicode.org/development/tests-and-tools). 
```

Comment:
This is probably good.

 However, it would be nice to improve the XSD generation to read attributes - then, there would be a broadly implemented way to verify some of these constraints (there's a ticket for this already.)